CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Improvements
 
 - ref: improve samples & add SPM docs ([#82](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/82))
+- ref: add documentation, detekt and dokka ([#86](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/86))
```

Comment:
@buenaflor changelog is meant for user-facing changes.

Reply:
Ah, I thought adding API documentation is part of user facing

Reply:
Yeah, it could be, detekt and dokka not really, but I agree, your call here.